mcux/drivers/imx/fsl_edma.c

```diff
@@ -1058,10 +1058,18 @@ status_t EDMA_SubmitTransfer(edma_handle_t *handle, const edma_transfer_config_t
             this check mechanism, EDMA_CreatHandle will clear CSR register.
         */
         tmpCSR = tcdRegs->CSR;
+        #if 0
```

Comment:
remove the dead code block and replace it with the fix. Please also update `mcux/README` to mention the change

Reply:
ok